frontend/helpers/next-helpers.ts

```diff
@@ -52,17 +52,10 @@ export const withNoindexDevDomains: GetServerSidePropsMiddleware = (next) => {
 };
 
 export const withSentry: GetServerSidePropsMiddleware = (next) => {
-   return (context) => {
-      return new Promise((resolve, reject) => {
-         next(context).then(resolve, (e) => {
-            Sentry.captureException(e);
-            console.log('Reporting exception to sentry');
-            console.log(e);
-            setTimeout(() => {
-               console.log('delayed for a bit, now carrying on');
-               reject(e);
-            }, 1000);
-         });
-      });
-   };
+   process.on('unhandledRejection', (reason) => {
```

Comment:
I think we shouldn't be doing this *inside* the `withSentry()` call. Otherwise, if you use `withSentry()` more than once it will subscribe multiple times. Vercel is kinda special in that they only run one next.js *page* per process, but not every environment is like that.

I think we should do it at the module level (`sentry.client.config` and `sentry.server.config`)

Reply:
Ah yea good call. This implementation didn't report to Sentry, nor did I see the logs, but I will try it in the config file.

Reply:
Womp womp still no dice when registered in `sentry.server.config`